packages/config/config/devices/0x0460/templates/wave_template.json

```
@@ -360,6 +360,74 @@
 		"maxValue": 32535,
 		"defaultValue": 30
 	},
+		"operating_modes": {
+		"label": "Operating Modes",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Shutter",
+				"value": 0
+			},
+			{
+				"label": "Venetian blind",
+				"value": 1
+			},
+			{
+				"label": "Set Manual time",
+				"value": 2
+			}
+		]
+	},
+		"motor_limit_detection": {
+		"label": "Motor Limit Detection",
+		"description": "Power threshold used to detect a limit switch. Increase this value if the device falsely detects limit switches.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Auto calibration",
+				"value": 1
+			}
+		]
+	},
+		"delay_limit_detection": {
+		"label": "Limit Detection Delay Time",
+		"description": "Defines the delay before motor power is read after a relay switches ON",
+		"unit": "0,1 sec",
+		"valueSize": 1,
+		"minValue": 3,
```

```suggestion
		"minValue": 0,
```
We currently have no good way to declare gaps in the valid value range. As a workaround, we use the description field (see comment above) to explain those.